Sources/TuistSupport/UI/UIController.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+import Noora
+
+/// A controller for UI elements on the terminal.
+public struct UIController: Noorable {
+    // MARK: - Properties
+
+    let noora: Noorable
+
+    /// Flag to silent all output
+    let isQuiet: Bool = ProcessInfo.processInfo.environment[Constants.EnvironmentVariables.quiet] != nil
```

Comment:
Once we move the responsibility to Noora, this information should be passed in `TuistCommand` when instance of `Noora` is created.

Reply:
Added a TODO for this as well